src/styles/components/buttons.less

```diff
@@ -35,6 +35,28 @@ components/buttons.less
         }
       }
     }
+
+    &-up {
+      &:after {
+        content: '';
+        border-bottom: 6px solid color-lighten(@neutral, 10);
+        border-left: 5px solid transparent;
+        border-right: 5px solid transparent;
+        vertical-align: middle;
+        display: inline-block;
+      }
+    }
+
+    &-down {
+      &:after {
+        content: '';
+        border-top: 6px solid color-lighten(@neutral, 10);
+        border-left: 5px solid transparent;
+        border-right: 5px solid transparent;
+        vertical-align: middle;
+        display: inline-block;
+      }
```

Comment:
Let's combine this with the above so that we're not repeating the same styles. Also :abcd:
```css
&-up,
&-before {
  &:after {
    // Shared styles
  }
}
```